src/python/pants/core/util_rules/archive_test.py

```diff
@@ -64,6 +65,19 @@ def test_extract_tar(rule_runner: RuleRunner, compression: str) -> None:
     assert digest_contents == EXPECTED_DIGEST_CONTENTS
 
 
+def test_extract_tarlz4(rule_runner: RuleRunner):
```

Comment:
We probably want to skip this if the `lz4` binary isn't on the system, right?

Reply:
Yea, probably... I don't have it on macOS 11.6.3.

Reply:
But do not skip it in CI

Reply:
@Eric-Arellano I'm saddened you even had to say anything ;)

Reply:
haha i was being lazy not finding your pr to point out Not An Issue :D